src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamsChangeListener.scala

```diff
@@ -116,7 +116,7 @@ object StreamsChangeListener {
           Path(settings.progressDir).deleteRecursively()
         }
       } catch {
-        case e => e.printStackTrace(System.err)
+        case e: Throwable => e.printStackTrace(System.err)
```

Comment:
Best practice is to catch `NonFatal` exceptions since exceptions are used for some control flow in Scala, and catching `Throwable` can actually break the semantics of the language.

```scala
import scala.util.control.NonFatal
...
try { f(x) }
catch { case NonFatal(t) => }
```